EIPS/eip-draft_reprice_opcodes.md

```diff
@@ -0,0 +1,145 @@
+---
+eip: <to be assigned>
+title: Opcode repricing 
+author: Martin Holst Swende (@holiman)
+discussions-to: https://ethereum-magicians.org/t/opcode-repricing/3024
+status: Draft
+type: Core
```

Comment:
This is missing a `category`.
See https://github.com/ethereum/EIPs/blob/master/eip-X.md
You probably want to use

```
type: Standards Track
category: Core
```

Reply:
The example says
```
category (*only required for Standard Track): <Core | Networking | Interface | ERC>
```
So I didn't add one

Reply:
That's an unexpected read of that template.
The way to read that is: `type` is always required, but the extra `category`-field is only required when you use `type: Standards Track`. I updated it for you.

Reply:
Ah I see, thanks!